packages/interactive/lib/src/executor.dart

```diff
@@ -83,9 +83,12 @@ class Executor {
     log.info('Phase: Evaluate');
     final isolateInfo = await workspaceIsolate.isolateInfo;
     final targetId = isolateInfo.rootLib!.id!;
-    final response = await vm.vmService
+    Response response = await vm.vmService
         .evaluate(workspaceIsolate.isolateId, targetId, _evaluateCode);
-
+    if(response is InstanceRef && response.valueAsString == null) {
```

Comment:
And please add a test in e2e_test.dart, which is quite simple - just provide some inputs and assert outputs